helsingborg-stad/aterbruket-webapp
# Bugfix/crash when create advert

**Explain the changes you’ve made**
removed some annoying console.logs 
also tried to add cleanups to useEffects because it is some warnings about that (not sure if it worked but thats another task)
 
**Explain why these changes are made**
Because the app crashes sometimes when creating an advert, or gives a warning-message
 
**Explain your solution**
Added a ternary on the department in addItem.tsx. to make sure the field is never uncontrolled, and always has a value. 
 
**How to test the changes?**
Try to create adverts and be aware if warnings appears. 
Also update the page when you're in add, just to see if that works, and not crashes
(this task does not solve the warning about useEffect when going in to the add-page)
